docs/messaging/ios-notification-images.md

```diff
@@ -68,6 +70,34 @@ At this point everything should still be running normally. This is the final ste
 
 ![step-3](/assets/docs/messaging/ios-notification-images-step-3.gif)
 
+### Step 3 - Use the extension helper (Swift)
+
+At this point everything should still be running normally. This is the final step which is invoking the extension helper.
+
+- From the navigator select your `ImageNotification` extension
+- Open the `NotificationService.swift` file
+- At the top of the file import `Firebase` right after the `NotificationService` as shown below
+
+```diff
+import UserNotifications
++ import Firebase
```

Comment:
Not 100% sure why the import here is `Firebase` instead of `FirebaseMessaging` - by following the steps in the doc, only `Firebase` is available.